data/mods/Magiclysm/monsters/Orcs.json

```diff
@@ -56,18 +62,19 @@
         { "item": "armguard_scrap", "prob": 40 },
         { "item": "cuirass_scrap", "prob": 40 },
         { "item": "longbow", "prob": 100 },
-        { "item": "sheath", "contents-item": "knife_combat", "prob": 50 }
+        { "item": "sheath", "contents-item": "knife_combat", "prob": 50 },
+        { "item": "quiver_large", "contents-group": "quiver_orc_archer", "prob": 100 }
       ]
     },
-    "starting_ammo": { "arrow_wood_heavy": 75 },
+    "starting_ammo": { "arrow_wood_heavy": 60 },
     "extend": {
       "special_attacks": [
         {
           "type": "gun",
           "cooldown": 10,
           "move_cost": 93,
           "gun_type": "bow",
-          "ammo_type": "arrow",
+          "ammo_type": "arrow_wood_heavy",
           "fake_skills": [ [ "gun", 6 ], [ "rifle", 7 ] ],
```

Comment:
I am not sure if `rifle` here should be changed to `archery`, changing it didn't seem to affect anything.

Reply:
I'll see about taking a look at the C++ code (which I should have done long ago... sigh!).

Reply:
```suggestion
          "fake_skills": [ [ "gun", 6 ], [ "archery", 7 ] ],
```